datatypes/tests/example-arrow.rs

```diff
@@ -722,3 +724,53 @@ fn float_equality() {
     assert_ne!(floats, floats3);
     assert_ne!(floats2, floats3);
 }
+
+#[test]
+fn filter_example() {
+    let a = Int32Array::from(vec![Some(1), Some(2), Some(3)]);
+
+    // dbg!(&a);
```

Comment:
delete the comments?!

Reply:
I left them on purpose since you can quickly uncomment them to try arrow stuff out and see debug output.
It is not a *regular* test but the arrow examples testbed.
But I can remove them if you want me to.